java-checks/src/test/java/org/sonar/java/checks/tests/IgnoredTestsCheckTest.java

```diff
@@ -33,4 +34,13 @@ void test() {
       .withCheck(new IgnoredTestsCheck())
       .verifyIssues();
   }
+
+  @Test
+  void test_without_semantic() {
+    JavaCheckVerifier.newVerifier()
+      .onFile(testSourcesPath("checks/IgnoredTestsCheck.java"))
+      .withCheck(new IgnoredTestsCheck())
+      .withClassPath(Collections.emptyList())
```

Comment:
Maybe we could consider at the end of this sprint adding a test that checks that all rules do not raise FP when the classPath is empty. It will help to not have to do this effort again in the future.
I'm a bit reluctant though because it will probably slow the (already slow) build quite a bit.
Let's keep the idea in mind and see if we can do something.